.github/workflows/testing.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           if-no-files-found: error
       - name: Run tests
         run: |
-          pytest --pyargs delocate --log-level DEBUG --cov-config=.coveragerc --cov=delocate --doctest-modules
+          pytest
```

Comment:
This test was always local since `--pyargs delocate` will import the local pacakge when run from the project directory.

`--pyargs delocate` becomes `testpaths` in the ` pyproject.toml` settings.  All of these settings were moved to that file.